xbmc/input/IRTranslator.cpp

```diff
@@ -130,6 +131,31 @@ void CIRTranslator::Clear()
   m_irRemotesMap.clear();
 }
 
+uint32_t CIRTranslator::TranslateButton(const TiXmlElement *pButton)
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types